src/libraries/System.Text.RegularExpressions/ref/System.Text.RegularExpressions.cs

```diff
@@ -287,4 +289,46 @@ public abstract partial class RegexRunnerFactory
         protected RegexRunnerFactory() { }
         protected internal abstract System.Text.RegularExpressions.RegexRunner CreateInstance();
     }
+    public enum RegexParseError
+    {
+        Unknown,
+        AlternationHasTooManyConditions,
+        AlternationHasMalformedCondition,
+        InvalidUnicodePropertyEscape,
+        MalformedUnicodePropertyEscape,
+        UnrecognizedEscape,
+        UnrecognizedControlCharacter,
+        MissingControlCharacter,
+        InsufficientOrInvalidHexDigits,
+        QuantifierOrCaptureGroupOutOfRange,
+        UndefinedNamedReference,
+        UndefinedNumberedReference,
+        MalformedNamedReference,
+        UnescapedEndingBackslash,
+        UnterminatedComment,
+        InvalidGroupingConstruct,
+        AlternationHasNamedCapture,
+        AlternationHasComment,
+        AlternationHasMalformedReference,
+        AlternationHasUndefinedReference,
+        CaptureGroupNameInvalid,
+        CaptureGroupOfZero,
+        UnterminatedBracket,
+        ExclusionGroupNotLast,
+        ReversedCharacterRange,
+        ShorthandClassInCharacterRange,
+        InsufficientClosingParentheses,
+        ReversedQuantifierRange,
+        NestedQuantifiersNotParenthesized,
+        QuantifierAfterNothing,
+        InsufficientOpeningParentheses,
+        UnrecognizedUnicodeProperty
+    }
```

Comment:
Can you add triple slash comments here? If you're on VS, if you type `///`, it'll auto-add a comment snippet. If not, it looks like this:
```      /// <summary>
         /// A parse exception containing the error and offset (feel free to rephrase if you want)
         /// </summary>
```

Reply:
Actually, the comments need to go in the src file, not in the ref :)

Reply:
@pgovind, yeah, that was an open question: what to do with the comments. I assume it should be added to each enum?

Reply:
I've added doc comments for each field.